tests/interlocked_int/interlocked_int.c

```diff
@@ -0,0 +1,855 @@
+//Copyright(c) Microsoft.All rights reserved.
+//Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifdef __cplusplus
+#include <cstdlib>
+#include <cstddef>
+#include <cstdint>
+#else
+#include <stdlib.h>
+#include <stddef.h>
+#include <stdint.h>
+#endif
+
+
+#include "testrunnerswitcher.h"
+
+static TEST_MUTEX_HANDLE g_testByTest;
+
+
+
```

Comment:
nitpick: You can collapse these empty lines into 1, same with removing the empty lines 31/32, 37, 47. (not a big deal)